src/modules/transaction/adapter/mapper/transaction.mapper.ts

```diff
@@ -0,0 +1,17 @@
+import { TransactionDatabaseModel } from '../../entity/transaction.database-model';
+import { TransactionOutput } from '../dto/transaction.output';
+
+export class TransactionMapper {
+    static mapToTransactionOutput(transaction: TransactionDatabaseModel): TransactionOutput {
+        return {
+            id: transaction?.id,
```

Comment:
@PCalape Is there a way to not use optional chaining to access the properties of `transaction`? Is there a chance that this will not have a value?

Reply:
Hello @arnellebalane my bad on this one, this was due to a refactoring move in which I did not continue. This should be required, will change this one. Thanks!

Reply:
Done updating this one @arnellebalane thanks!